docs/pages/enroll-resources/desktop-access/directory-sharing.mdx

```diff
@@ -58,7 +57,14 @@ and click "Share Directory":
 
 #### Prohibited directories
 
-For security reasons the web browser's filesystem access API prohibits you from
+<Admonition type="note">
```

Comment:
I would:
- Put the `Admonition` at the end of the section, since readers won't know what "this restriction" refers to until they read the text
- Add "(Web UI only)" in the heading so Connect users can skip it without needing to become familiar with the content (the heading would be, `#### Prohibited directories (Web UI only)`

Reply:
Thanks, done.